### DISCUSSION TITLE
Create Discord context menu function?

### DISCUSSION THREAD

**Discussion by enter9991** [discord-php/DiscordPHP#1167] [Q&A] [answered]
Hello, I wanted to ask if there is a function to create a context menu in Discord. Unfortunately, I haven't found it or simply overlooked it.

https://discord.com/developers/docs/interactions/application-commands

**Comment by key2peace** [comment, marked as answer]
All you need to do is set the type to either ``Command::USER`` or ``Command::MESSAGE``
